portfolio/src/main/webapp/contact.css

```diff
@@ -0,0 +1,83 @@
+html, body {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+}
+
+#header {
+    display: flex;
+    flex-direction: row;
+    padding: 10px;
+    align-items: center;
+    background-color: rgb(192, 192, 192);
+    background-image: linear-gradient(rgb(192, 192, 192), rgb(129, 129, 129));
+}
+
+#header img {
+    height: 75px;
+    width: 100px;
+}
+
+.contact_content {
+    min-height: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    background-color: black;
+}
+
+#message {
+    flex-grow: 1;
+}
+
+#message, #footer {
+    flex-shrink: 0;
+}
+
+#message {
+    align-content: center;
+    background-color: yellow;
+    width: 100%;
+}
```

Comment:
Again, please alphabetize.. Including the nested tags.

Reply:
alphabetized